aten/src/TH/THAllocator.c

```diff
@@ -478,6 +539,18 @@ static void * THRefcountedMapAllocator_alloc(void *_ctx, ptrdiff_t size) {
   char *data = ((char*)ptr) + TH_ALLOC_ALIGNMENT;
   THMapInfo *map_info = (THMapInfo*)ptr;
 
+#ifdef _WIN32
+  ReleaseContext* cxt = (ReleaseContext *) THAlloc(sizeof(ReleaseContext));
+  cxt->handle = ctx->handle;
+  cxt->event = ctx->event;
+  cxt->data = (void*)data;
+  HANDLE thread = CreateThread(NULL, 0, WaitForReleaseHandle, (LPVOID)cxt, 0, NULL);
```

Comment:
Ouch, how expensive if `CreateThread`? Having a *thread per tensor* sounds a bit prohibitive. Isn't there a way to have a single thread with a queue that's polling on a number of events at the same time?

Reply:
Yes, it's quite expensive on Windows. I'll try to keep only one daemon thread for one process but I don't know which part of this file only run once. Otherwise, a flag will be needed.

Reply:
Sorry, I've reconsidered this. It's extremely difficult to implement this. There're mainly two difficulties, synchronization and queue implementation.
1. Because we have to send the handles to the queue but for both processes the handles are actually different. We'll need to link the `Event` with the `FileMapping`, but the `Event` may be blocked since the sequence of freeing is not strictly synchronized. How can we resolve this? 
2. It seems that we don't have an existing structure of `Queue` in `TH`. Shall we implement it?